pkg/pillar/cmd/zedagent/handlecertconfig.go

```diff
@@ -193,3 +193,19 @@ func unpublishControllerCert(ctx *getconfigContext, key string) {
 	log.Debugf("unpublishControllerCert %s Done\n", key)
 	pub.Unpublish(key)
 }
+
+func handleAttestCertModify(ctxArg interface{}, key string,
+	configArg interface{}) {
+
+	status := configArg.(types.AttestCert)
+	log.Infof("handleAttestCertModify for %s\n", status.Key())
+	return
+}
+
+func handleAttestCertDelete(ctxArg interface{}, key string,
+	configArg interface{}) {
+
+	status := configArg.(types.AttestCert)
+	log.Infof("handleAttestCertDelete for %s\n", status.Key())
+	return
+}
```

Comment:
Are you/Srinibas planning to hook in code here later? If so might make sense to add some XXX TBD comment about it.

Reply:
@srinibas-zededa is planning to hook in code here. I will add the necessary comments.